### DISCUSSION TITLE
Webhook/push support

### DISCUSSION THREAD

**Discussion by rusty1281** [lukevella/rallly#980] [Ideas]
Hi,

would webhook support be of any use for this platform? For example, notifications that a new comment has been published, or that a participant has chosen a date? I know that getting a final email with an ICS makes more sense to be delivered via email, but all the notifications before that might be routed via webhook or push service.

Just an idea!

Tnx for a great platform!

**Comment by satonotdead** [comment]
I was thinking about an integration with automation services like N8N or ActivePieces. That should add a lot of value to Rallly :)
